source/libsass.html.haml

```diff
@@ -37,11 +37,17 @@ title: libSass
   %li#go
     :markdown
       ### Go
-      [Wellington](https://github.com/wellington/wellington) is an extension to libsass that adds spriting and is available on brew:
-
-      `brew install wellington`
-
-      There are also three other libsass wrappers in go: [gosass](https://github.com/moovweb/gosass), [go-sass](https://github.com/SamWhited/go-sass) and [go_sass](https://github.com/suapapa/go_sass) which have not been updated in a while.
+      [go-libsass](https://github.com/wellington/go-libsass) has the most active
```

Comment:
The last commit to go-libsass is almost a year ago. There have been several major LibSass releases since then. IMHO this project looks to be abandoned.